src/integrationTest/resources/hibernate.properties

```diff
@@ -1,3 +1,3 @@
-jakarta.persistence.jdbc.url=mongodb://localhost/mongo-hibernate-test?directConnection=false
 hibernate.dialect=com.mongodb.hibernate.dialect.MongoDialect
 hibernate.connection.provider_class=com.mongodb.hibernate.jdbc.MongoConnectionProvider
+jakarta.persistence.jdbc.url=mongodb://localhost/mongo-hibernate-test?directConnection=false
```

Comment:
I rearranged the lines in the order of their importance: the dialect is and the provider are the most important ones, as they plug our product into the Hibernate ORM framework. `jakarta.persistence.jdbc.url` is the next most important configuration property.